src/modules/app/scanner.js

```diff
@@ -19,7 +19,7 @@ const keyPressTimer$ = keyPressReducer$.debounce(() => Observable.timer(50))
     return {...state, buffer: '', code: buffer, uuid};
```

Comment:
If you have time then you can take this out in Util and create test-case for UUID.